app/javascript/controllers/app/search_controller.js

```diff
@@ -24,6 +25,9 @@ export default class extends Controller {
       {{/results}}
     </ul>
   `
+
+    this.throttledAutocomplete = _.throttle(this.autocomplete, 300)
```

Comment:
Your recent changes look good, but you'll need to call throttle as the imported function `throttle`, instead of referencing it through `_`. Otherwise you get:

```
Error initializing controller
ReferenceError: Can't find variable: _
```

```suggestion
    this.throttledAutocomplete = throttle(this.autocomplete, 300)
```

Reply:
Whoops, that's what I get for pushing changes without testing locally. Fixed in d09d7d515991476a9de095cfd0516335ab0d134c